async_processor/nats_pub_sub.py

```diff
@@ -93,23 +93,22 @@ async def get_input_message(
         self,
     ) -> AsyncIterator[Optional[bytes]]:
         psub = await self._get_psub()
-        while True:
+        msgs = []
+        try:
```

Comment:
will this context manager return more than one message??

Reply:
ohh got it, we are looping in the worker itself

Reply:
Yes, this was an over-complication. The worker loop will be the only loop for now.